OWNERS

```diff
@@ -0,0 +1,27 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
need to be updated to match the newest content in https://github.com/pingcap/community/blob/master/teams/kubernetes/membership.json, right?

Reply:
yes, you are right, the PR is outdated, I will update it.

Reply:
updated